get_current_tidepool_repos.sh

```diff
@@ -41,7 +41,20 @@ get_one_tidepool_repo()
     fi
 }
 
+setup_platform()
+{
+    echo "*** platform ***"
+    mkdir -p platform
+    export GOPATH=${PWD}/platform
+    # Ignore the "no buildable Go source files" warning
+    go get github.com/tidepool-org/platform 2>&1 | grep -v "no buildable Go source files"
```

Comment:
I'm not really in a position to answer that, so I'll let you make the call.
I was just following the initial process outlined in the platform README.md.